### ISSUE TITLE
Security issue when using ffmpeg-platform

### ISSUE THREAD

**Issue by zicjiang** [bytedeco/javacpp-presets#1253]
Hi, I'm using the ffmpeg-platform of version 5.0-1.5.7. but when I add this dependency into our project, some security issues had been detected by Corona. 
![image](https://user-images.githubusercontent.com/102562667/198923060-8ddaeab0-995f-4076-91e9-c4ac60b0bb21.png)
I checked into this issue and found some .so files which ffmpeg-platform brought in have used the risky version of openssl and zlib. how can we solve these issue? Thanks for your advice!

**Comment by saudet** [comment]
Could you please give it a try with the snapshots? http://bytedeco.org/builds/

**Comment by zicjiang** [comment]
Hi @saudet. this issue is still there. take the openssl as an example, it seems that the libavformat.so.59(linux version) which the dependency introduced has been using the openssl 1.1.1m 
![image](https://user-images.githubusercontent.com/102562667/198978756-167ba5ec-e10f-4b2a-94ce-4d5b454b4ca3.png)

**Comment by saudet** [comment]
Not seeing that here:
```
$ wget https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/ffmpeg/5.1.2-1.5.8-SNAPSHOT/ffmpeg-5.1.2-1.5.8-20221028.165821-109-linux-x86_64-gpl.jar
$ unzip ffmpeg-5.1.2-1.5.8-20221028.165821-109-linux-x86_64-gpl.jar
$ strings org/bytedeco/ffmpeg/linux-x86_64-gpl/* | grep OpenSSL
OpenSSL X9.42 DH method
OpenSSL PKCS#3 DH method
OpenSSL DH Method
OpenSSL DSA method
OpenSSL EC algorithm
OpenSSL EC_KEY method
OpenSSL ED448 algorithm
OpenSSL ED25519 algorithm
OpenSSL X448 algorithm
OpenSSL X25519 algorithm
%s:%d: OpenSSL internal error: %s
OpenSSL RSA-PSS method
OpenSSL RSA method
OpenSSL PKCS#1 RSA
OpenSSL NULL UI
OpenSSL default user interface
OpenSSL Base Provider
OpenSSL Default Provider
OpenSSL DSA implementation
OpenSSL RSA implementation
OpenSSL EC implementation
OpenSSL X25519 implementation
OpenSSL X448 implementation
OpenSSL ED448 implementation
OpenSSL SM2 implementation
OpenSSL PKCS#3 DH implementation
OpenSSL X9.42 DH implementation
OpenSSL RSA-PSS implementation
OpenSSL ED25519 implementation
OpenSSL TLS1_PRF via EVP_PKEY implementation
OpenSSL HKDF via EVP_PKEY implementation
OpenSSL SCRYPT via EVP_PKEY implementation
OpenSSL HMAC via EVP_PKEY implementation
OpenSSL SIPHASH via EVP_PKEY implementation
OpenSSL POLY1305 via EVP_PKEY implementation
OpenSSL CMAC via EVP_PKEY implementation
OpenSSL Null Provider
OpenSSL 'dlfcn' shared library method
OpenSSL NIST SP 800-90A DRBG
OpenSSL default
```

**Comment by zicjiang** [comment]
sorry. I searched the wrong version. but even though I used the version you suggested(I used the ffmpeg-5.1.1-1.5.8-SNAPSHOT-linux-x86_64.jar), it's still warning that the libavformat.so.59 is using the OpenSSL 1.1.1m. 
![image](https://user-images.githubusercontent.com/102562667/199131931-900a5cb8-2724-48d2-9ff7-a76cbf9d2691.png)
![image](https://user-images.githubusercontent.com/102562667/199132084-12c579c8-a773-43fe-a655-bd5727ced647.png)
so my question is does the libavformat.so.59 use any openssl 1.1.1m functions?
